app/models/validations/date_validations.rb

```diff
@@ -68,20 +68,32 @@ def validate_startdate(record)
 
 private
 
-  def first_collection_start_date
-    @first_collection_start_date ||= FormHandler.instance.lettings_forms["previous_lettings"].start_date
+  def previous_collection_start_suffix
+    previous_collection_start_date.strftime("%y").to_i
   end
 
-  def first_collection_end_date
-    @first_collection_end_date ||= FormHandler.instance.lettings_forms["previous_lettings"].end_date
+  def current_collection_start_suffix
+    current_collection_start_date.strftime("%y").to_i
```

Comment:
```suggestion
    current_collection_start_date.year
```

Reply:
`strftime("%y")` gives me e.g. 22 from 2022 which is what I want - I could do` .year % 100` I suppose, that's probably clearer

Reply:
`strftime("%y")` gives me e.g. 22 from 2022 which is what I want - I could do` .year % 100` I suppose, that's probably clearer